CHANGELOG.md

```diff
@@ -6,6 +6,82 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.7.0] 2019-03-26
+
+### TODO: Don't commit this without proofreading!!!
+
+### Added
+
+### Changed
+
+- Merge pull request #669 from cyberark/doodlesbykumbi-patch-1
```

Comment:
This needs to be cleaned up. Aggregating and summarizing things based on the commit history is fine, and even preferable. There should be no `Merge pull request` lines.